File: Cineplus/Models/Ticket.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Cineplus.Models
{
    public class Ticket : DbEntity
    {
        public int SeatId { get; set; }
        public virtual Seat Seat { get; set; }

        public string UserId { get; set; }
        public virtual ApplicationUser User { get; set; }

        public double Price { get; set; }

        public DateDiscount DateDiscount { get; set; }

        public virtual List<PersonalDiscount> PersonalDiscounts { get; set; }

        public double PointsPrice { get; set; }

        public Guid OrderId { get; set; }
        [JsonIgnore] public Guid Confirmation { get; set; }
        public DateTime ReserveTime { get; set; }
        public int ReproductionId { get; set; }

        public virtual Reproduction Reproduction { get; set; }
    }
}

File: Cineplus/ClientApp/src/models/ticket.ts
import {Seat} from "./seat";
import {Reproduction} from "./reproduction";
import {User} from "./user";
import {DateDiscount} from "./dateDiscount";
import {PersonalDiscount} from "./personalDiscount";

export class Ticket {
  seatId: number;
  seat: Seat;
  reproductionId: number;
  reproduction : Reproduction;
  userId: string;
  user: User;
  price: number;
  pointsPrice: number;
  dateDiscount: DateDiscount;
  personalDiscounts: PersonalDiscount[];
  orderId:string;
  reserveTime:Date;
}
